ear/src/main/resources/s3binaries/nuxeo.defaults

```diff
@@ -12,3 +13,10 @@ nuxeo.s3storage.crypt.key.alias=
 nuxeo.s3storage.crypt.key.password=
 
 nuxeo.core.binarymanager=org.nuxeo.ecm.core.storage.sql.S3BinaryManager
+
+nuxeo.s3storage.useDirectUpload=true
```

Comment:
Should it be true by default?

Reply:
I believe it should, since it has advantages without drawbacks (I can see, at least)

Reply:
It requires additional config, so let's set it `false` by default.